### PR TITLE
Droth 1255 new form to add oth maintainers

### PR DESCRIPTION
**Pull request by oskarmalmstrom**

### CODE REVIEW

File: UI/newuser.html [hunk 1]
```diff
@@ -56,61 +191,174 @@
...
+        function anyBoxChecked() {
+          return _.some([operatorCheckbox, municipalityCheckbox, elysCheckbox, areasCheckbox], function(checkbox){return $(checkbox).is(':checked');});
+        }
+
+        jQuery(document).ready(function($) {
+            $('#municipalities_search').multiselect({
+                search: {
+                    left:
+                        '<label class="control-label labelBoxLeft">Kaikki kunnat</label>' +
+                        '<input type="text" name="from[]" id = "left_municipalities" class="form-control" placeholder="Kuntanimi" style="width: 151px;" />',
```
Comment (joanacoelho) [inline UI/newuser.html]: the attribute name can be removed from here

File: UI/newuser.html [hunk 2]
```diff
@@ -56,61 +191,174 @@
...
+                    right:
+                        '<label class="control-label labelBoxRight">Valitut Kunnat</label>' +
+                        '<input type="text" name="to[]" id = "right_municipalities" class="form-control" placeholder="Kuntanimi" style="width: 151px;" />'
```
Comment (joanacoelho) [inline UI/newuser.html]: the attribute name can be removed from here

File: digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/MunicipalityService.scala [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+import fi.liikennevirasto.digiroad2.service.RoadLinkService
+
+class MunicipalityService (eventbus: DigiroadEventBus, roadLinkService: RoadLinkService) {
+
+  def withDynTransaction[T](f: => T): T = OracleDatabase.withDynTransaction(f)
+  def withDynSession[T](f: => T): T = OracleDatabase.withDynSession(f)
+
+  def getMunicipalities: Seq[(Int, String)] = {
+    withDynSession {
+      Queries.getMunicipalitiesInfo
```
Comment (MAC-CGI) [inline digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/MunicipalityService.scala]: Same method exist on municipalityDAO **getMunicipalitiesNameAndIdByCode(Set())**

File: digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/queries.scala [hunk 4]
```diff
@@ -250,6 +250,12 @@ object Queries {
     """.as[Int].list
   }
 
+  def getMunicipalitiesInfo: Seq[(Int, String)] = {
+    sql"""
+      select id, name_fi from municipality
+    """.as[(Int, String)].list
+  }
```
Comment (MAC-CGI) [inline digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/queries.scala]: Already exist   def getMunicipalitiesNameAndIdByCode(codes: Set[Int]): List[(Int, String)] = {
    val filter = if (codes.nonEmpty) {"where id in (" + codes.mkString(",") + ")" } else ""

    sql"""
      select id, name_fi from municipality
      #$filter
    """.as[(Int, String)].list
  } on municipalityDAO

The method bottom **getMunicipalitiesWithoutAhvenanmaa** could be moved to municipalityDAO as well

Comment (joanacoelho) [reply digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/queries.scala]: That method receives a parameter, The one that was created doesn't

File: UI/newuser.html [hunk 5]
```diff
@@ -16,21 +16,21 @@
...
-                     {name: "Lapin", id: 1},
-                     {name: "Pohjois-Pohjanmaan", id: 2},
-                     {name: "Etelä-Pohjanmaan", id: 3},
-                     {name: "Keski-Suomen", id: 4},
-                     {name: "Pohjois-Savon", id: 5},
-                     {name: "Pirkanmaan", id: 6},
-                     {name: "Varsinais-Suomen", id: 7},
-                     {name: "Kaakkois-Suomen", id: 8},
-                     {name: "Uudenmaan", id: 9}];
+        var elys = [ {name: "Ahvenanmaa", id: 0},
```
Comment (joanacoelho) [inline UI/newuser.html]: This elys names are not written according to the operators manual

File: UI/newuser.html [hunk 6]
```diff
@@ -3,16 +3,150 @@
...
-            else
-                $(".authArea").hide();
+        var operatorCheckbox = "input[id='operatorCheckbox']";
+        var municipalityCheckbox = "input[id='municipalitiesCheckbox']";
+        var elysCheckbox = "input[id='elysCheckbox']";
+        var areasCheckbox = "input[id='areasCheckbox']";
+        var municipalities;
+        var roleName = [];
+        var areas = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+        var elys = [ {name: "Ahvenanmaa", id: 0},
```
Comment (joanacoelho) [inline UI/newuser.html]: This elys names are not written according to the operators manual

**Pull request merged**